extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonUtils.java

```diff
@@ -23,16 +28,22 @@
 import io.smallrye.jwt.build.JwtSignatureBuilder;
 import io.smallrye.jwt.util.KeyUtils;
 import io.smallrye.jwt.util.ResourceUtils;
+import io.smallrye.mutiny.Uni;
 import io.vertx.core.http.HttpClientOptions;
 import io.vertx.core.json.JsonObject;
 import io.vertx.core.net.ProxyOptions;
 import io.vertx.mutiny.core.MultiMap;
 import io.vertx.mutiny.core.buffer.Buffer;
+import io.vertx.mutiny.ext.web.client.WebClient;
 
 public class OidcCommonUtils {
+    public static final Duration CONNECTION_BACKOFF_DURATION = Duration.ofSeconds(2);
```

Comment:
Wondering if people are not going to ask for making this configurable. But I guess you want to wait for it.

Reply:
@pedroigor yeah; it was like this from the very early days of `quarkus-oidc`, and no one asked for the delay between retries be configurable, I can surely make it configurable if someone finds it can be useful :-).